README.md

```diff
@@ -291,6 +291,14 @@ $ lerna run build
 
 Run an [npm script](https://docs.npmjs.com/misc/scripts) in each package that contains that script.
 
+### Import
+
+```sh
+$ lerna import <path-to-external-repository>
+```
+
+Import the package at `<path-to-external-repository>`, with commit history.
```

Comment:
If we want to be more verbose here, you can mention that the package x is imported into `packages/x` and maybe an example. You can even use lerna-changelog as the example?